src/platform/test/functional/page_objects/unified_tabs.ts

```diff
@@ -61,8 +61,8 @@ export class UnifiedTabsPageObject extends FtrService {
 
   public async createNewTab() {
     const numberOfTabs = await this.getNumberOfTabs();
-    await this.testSubjects.click('unifiedTabs_tabsBar_newTabBtn');
     await this.retry.waitFor('the new tab to appear', async () => {
+      await this.testSubjects.click('unifiedTabs_tabsBar_newTabBtn');
```

Comment:
@akowalska622 I thought we don't need to retry `click` with `getPreviewData` being memoized, do we?

Reply:
Fixed in [c0f17bd](https://github.com/elastic/kibana/pull/219059/commits/c0f17bdff296e1a3be2fe15d488a11718497601e)